.pre-commit-config.yaml

```
@@ -30,3 +30,8 @@ repos:
   rev: '3.9.2'
   hooks:
   - id: flake8
+
+- repo: https://github.com/asottile/reorder_python_imports
+  rev: v3.8.3
+  hooks:
+  - id: reorder-python-imports
```

Please use `isort` instead, if you're using an import statement sorter.
```suggestion
  - repo: https://github.com/timothycrosley/isort
    rev: 5.10.1
    hooks:
    - id: isort
      args: ["--profile", "black", "--filter-files", "--skip-gitignore"]
```